agent/src/index.ts

```diff
@@ -81,7 +81,6 @@ import { webSearchPlugin } from "@elizaos/plugin-web-search";
 
 import { giphyPlugin } from "@elizaos/plugin-giphy";
 import { letzAIPlugin } from "@elizaos/plugin-letzai";
-import { thirdwebPlugin } from "@elizaos/plugin-thirdweb";
```

Comment:
was this intentional?

Reply:
No, it was deleted by mistake, I changed it back.